include/caf/sensor_sampler.h

```diff
@@ -13,6 +13,26 @@ extern "C" {
 
 #include <drivers/sensor.h>
 
+enum trigger_activation_type {
+	PERCENTAGE_DEVIATION,
```

Comment:
Please add prefix for enum values matching enum name
`TRIGGER_TYPE_PERCENTAGE`
`TRIGGER_TYPE_ABSOLUTE`
?